api/v1/policyset_types.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PolicySetSpec defines the desired state of PolicySet
+type PolicySetSpec struct {
+	Description string `json:"description,omitempty"`
+	// +kubebuilder:validation:Required
+	Policies []string `json:"policies"`
+}
+
+// PolicySetStatus defines the observed state of PolicySet
+type PolicySetStatus struct {
+	Placement []PolicySetStatusPlacement `json:"placement"`
```

Comment:
`omitempty` is missing which makes the field required. You may run into some issues. In general, `status` field shouldn't be required.